flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogMapStateTest.java

```diff
@@ -36,80 +36,80 @@
 
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.singletonMap;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** ChangelogMapState Test. */
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ChangelogMapStateTest {
```

Comment:
```suggestion
class ChangelogMapStateTest {
```
could be package private